kyaml/kio/byteio_reader.go

```diff
@@ -101,8 +101,16 @@ type ByteReader struct {
 	// WrappingKind is set by Read(), and is the kind of the object that
 	// the read objects were originally wrapped in.
 	WrappingKind string
+
+	// AcceptJSON indicates if the input json bytes should be processed
+	AcceptJSON bool
```

Comment:
Why is this necessary?

Reply:
To reduce the blast radius, wanted to introduce it only to source command. We can get rid of it eventually when we want to roll out to all commands.